Add project description for llm benchmark (#119)
MinBZK/ai-validation: docs/Projects/LLM-benchmarks/project_description.md

> # LLL-Benchmark project

Reviewer: ```suggestion
# LLM-Benchmark project
```

> 

Reviewer: ```suggestion
## Context
```

> Large Languages Models (LLM's) are becoming increasingly popular in assisting people in

Reviewer: ```suggestion
Large Languages Models (LLMs) are becoming increasingly popular in assisting people in
```

Reviewer: I think this is the more used plural form of LLM

a variety of tasks. These tasks include, but are not limited to, information retrieval, assisting with

> coding and essay writing. In the context of the government, tasks can include for example supporting FOIA

Reviewer: I think the abbreviation FOIA is maybe not so well known and therefore weird to introduce it here in one go maybe

Reviewer 2: Agreed, I would have to Google it

requests and aiding in answering questions of citizens.

> While the potential benefit of using LLM's is large, there are also significant risks.

Reviewer 2: ```suggestion
While the potential benefit of using LLMs is large, there are also significant risks.
```

> Basically an LLM is just a next token predictor, which bases its predictions on the user input (context) and

Reviewer: I like it, but at the same time I am doubting why specific technologies should be introduced for the project itself, just putting my thought out there

Reviewer 3: What do you mean with specific technologies?

> Basically an LLM is just a next token predictor, which bases its predictions on the user input (context) and

Reviewer 2: ```suggestion
Basically a LLM is just a next token predictor, which bases its predictions on the user input (context) and
```

Reviewer 3: Wait, I think it should be "an LLM" since LLM is pronounced as "el-el-em".

on compressed information seen during training (LLM parameters); hence there
is no guarantee on the quality and correctness of the output. Moreover, due to

> bias in the training data, LLM's can have bias in their output, despite best efforts

Reviewer 2: ```suggestion
bias in the training data, LLMs can have bias in their output, despite best efforts
```

> to mitigate this. Additionally, we have human values that we expect LLM's to be aligned with.

Reviewer 2: ```suggestion
to mitigate this. Additionally, we have human values that we expect LLMs to be aligned with.
```

Certainly, within the context of a government, we should take utmost care not to discriminate.
To assess the quality, correctness, bias and alignment with human values of a LLM one can perform
benchmarks.

## The project

> The LLM-benchmark project of the AI-Validation Team of the Ministry of the Interior and Kingdom

Reviewer: ```suggestion
The LLM-benchmark project of the AI-Validation Team 
```

Reviewer: This is implied by this being in this repository and also is just terribly long

> Relations of the Government of the Netherlands aims to create a platform where LLM's can be measured

Reviewer: ```suggestion
 aims to create a platform where LLMs can be measured
```

> across a wide range of benchmarks. Both LLM's and the benchmarks can be configured by users of the platform.

Reviewer: I miss the piece here still what the project is exactly, you state here users can configure them, this does imply that it is some kind of platform where they can be ran.

> across a wide range of benchmarks. Both LLM's and the benchmarks can be configured by users of the platform.

Reviewer 2: ```suggestion
across a wide range of benchmarks. Both LLMs and the benchmarks can be configured.
```

The intended goal of this project is to give government organizations, citizens and companies insight in the various

> LLM's and their quality, correctness, bias and alignment with human values. The project also encompasses a dashboard with

Reviewer 2: ```suggestion
LLMs and their quality, correctness, bias and alignment with human values. The project also encompasses a dashboard with
```

> uploaded LLM's and their performance on uploaded benchmarks. With this platform we aim to enhance

Reviewer 2: ```suggestion
uploaded LLMs and their performance on uploaded benchmarks. With this platform we aim to enhance
```

> public trust in the usage of LLM's and expose potential bias that exists within LLM's.

Reviewer 2: ```suggestion
public trust in the usage of LLMs and expose potential bias that exists within LLMs.
```
